certified-connectors/Bitvore/apiDefinition.swagger.json

```diff
@@ -0,0 +1,773 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "The Entity API provides access to the information aboutcorporate entities. \n\n Companies can be searched for based on some criteria or directly using a Bitvore ID which uniquely identifies an entity. When using the Bitvore ID the entire company profile will be returned for that single company. In addition, a search can be performed using different profile properties such as name, ticker, industry, etc. The API will return all companies that match the search criteria.",
+    "version": "2.0",
+    "title": "Bitvore Corporate Entity",
```

Comment:
Is the title meant to be `Bitvore Corporate Entity` or `Bitvore Cellenus Entity`? In the readme it is referred to Cellenus entity, but here it is referred to as Corporate.

Reply:
updated both to be Bitvore Entity API

Reply:
@eriklg Unfortunately the term 'API' isn't allowed in the title of the connector. It can be used elsewhere, such as the description, but just not within the title. So something like `Bitvore Entity` would be perfectly acceptable if that works for you.

Reply:
Pushed!